src/components/stub/stub.tsx

```diff
@@ -16,15 +16,28 @@
 import 'twin.macro';
 
 interface Props {
+  className?: string;
   children: React.ReactNode;
+  title: React.ReactNode;
+  message: React.ReactNode;
 }
 
-export const NoResultsFoundSub = ({ children }: Props) => (
-  <div tw="grid place-items-center mt-21 text-monochrome-medium-tint">
-    {children}
-    <div tw="mt-4 mb-2 text-20 leading-32 text-monochrome-default">No results found</div>
-    <div tw="text-14 leading-20 text-monochrome-default text-center">
-      Try adjusting your search or filter to find what you are looking for.
+export const Stub = ({
+  children, title, message, className,
+}: Props) => (
+  <div tw="flex flex-col justify-center items-center w-full h-full flex-grow text-monochrome-medium-tint py-10" className={className}>
```

Comment:
We have a semantic block, can we create a StyledElement?
```suggestion
  const Icon = styled.div``
```

Reply:
@RomanAchilov I would argue against that approach.
It's the only place where these particular classes are used. I don't see a point in hiding them behind another opaque wrapper that makes code harder to read.